packages/terra-framework-docs/src/terra-dev-site/test/modal-manager/DisclosureIntegrationComponent.jsx

```diff
@@ -139,7 +139,7 @@ class DisclosureComponent extends React.Component {
           {this.renderFormButton()}
           <br />
           <br />
-          <SelectField label="T-shirt size" placeholder="Select a size" selectId="tshirt-size-field" required className={cx('select-field')}>
+          <SelectField variant="combobox" label="T-shirt size" placeholder="Select a size" selectId="tshirt-size-field" required className={cx('select-field')}>
```

Comment:
@AH106586Harika have you created separate defect to fix the issue of terra-form-single-select not working within modal ..??

Reply:
Raised Defect for form-single-select: [UXPLATFORM-10169](https://jira2.cerner.com/browse/UXPLATFORM-10169)